### PR TITLE
edit command palette topic

### PR DESCRIPTION
**Pull request by guillermooo**
@FichteFoll

### CODE REVIEW

File: source/extensibility/command_palette.rst [hunk 1]
```diff
@@ -12,48 +12,42 @@ Command Palette
...
-key binding of their own are good candidates for inclusion in a ``.sublime- commands``
-files.
+The *command palette* is an interactive list
+whose purpose is to execute commands.
+The command palette is fed entries
+by means of ``.sublime-commands`` files.
+Commands that don't warrant
+creating a key binding of their own
+are usually good candidates
+for inclusion in a ``.sublime- commands`` file.
```
Comment (FichteFoll) [inline source/extensibility/command_palette.rst]: extra space here

File: source/extensibility/command_palette.rst [hunk 2]
```diff
@@ -12,48 +12,42 @@ Command Palette
...
 
-By default, the command palette includes many useful commands, and provides
-convenient access to individual settings as well as settings files.
+By default,
+the command palette includes many useful commands,
```
Comment (FichteFoll) [inline source/extensibility/command_palette.rst]: remove comma?

File: source/extensibility/command_palette.rst [hunk 3]
```diff
@@ -12,48 +12,42 @@ Command Palette
...
+and provides convenient access
+to individual settings as well as settings files.
 
 
-File Format (Commands Files)
-============================
+File Format of ``.sublime-commands`` Files
```
Comment (FichteFoll) [inline source/extensibility/command_palette.rst]: Imo, the file format should probably go into the reference section entirely. But we'll talk about that in the restructure process. It's there already, so no need to fuss about it in this particular PR.

File: source/extensibility/command_palette.rst [hunk 4]
```diff
@@ -12,48 +12,42 @@ Command Palette
...
-``caption``
-   Text for display in the command palette.
-``command``
-   Command to be executed.
-``args``
-   Arguments to pass to ``command``.
+.. include:: ../common/command-palette-example-0.txt
 
 
 How to Use the Command Palette
```
Comment (FichteFoll) [inline source/extensibility/command_palette.rst]: I suggest moving this very important part of the page higher up. And adding a screenshot/gif.

File: source/common/command-palette-table-summary.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
+=============  =============================================================
+**Format**     JSON (with comments)
+
+**Extension**  ``.sublime-commands``
+
+**Name**       Any
+
+**Location**   Any under :file:`{Packages}`
+
+**Content**    A list of :ref:`Command Palette Item`\s
```
Comment (FichteFoll) [inline source/common/command-palette-table-summary.txt:10]: How does this backslash render?

Comment (guillermooo) [inline source/common/command-palette-table-summary.txt:10]: Fine. Without it it wouldn't work IIRC.

Comment (guillermooo) [inline source/common/command-palette-table-summary.txt:10]: I mean, it doesn't render in the final output :)

Comment (FichteFoll) [inline source/common/command-palette-table-summary.txt:10]: Good. I think I used/saw this somewhere else but don't remember anymore.

**Pull request merged**